CONTRIBUTING.md

```diff
@@ -40,8 +40,7 @@ At this point, you can start running the bot.
 
 1. Locate your discord bot token from the Bot screen
 2. Define `HOJBOT_DISCORD_TOKEN` environment variable in your shell profile.
-3. Define `FINNHUB_TOKEN` environment variable if you are developing for the investment game. You can get your own [API key from finnhub.io](https://finnhub.io/).
-4. Start the bot using `script/run.sh` script
+3. Start the bot using `script/run.sh` script
```

Comment:
It might be interesting to add the comment that, for prototyping other features, one can start with `ENABLE_WARM_UP=0 script/run.sh`
```suggestion
3. Start the bot using either `script/run.sh` script, or `ENABLE_WARM_UP=0 script/run.sh`. The latter is useful if you want to test features other than the investment game and don't want to wait for the long warm up period needed for setting it up.
```